_docs/migrate_module.md

```diff
@@ -12,11 +12,13 @@ You will have someone by your side in this process. The general flow is to…
 * Ensure github issues are enabled.
 * Verify that all webhooks are disabled.
 * Add the module to our [modulesync setup][managed_modules].
+* Enable `Automatically delete head branches` in the repository settings.
 * Ask an admin to add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration) (that link works only for admins).
 * The admin shall also update the [access permissions](https://github.com/organizations/voxpupuli/settings/secrets/actions) (that link works only for admins) for forge.puppet.com secrets so releases can be published.
 * The admin shall also enable `Automatically delete head branches` in the repository settings.
 * Execute [modulesync][msync] for this module.
-* Create a Jira issue at [tickets.puppetlabs.com](https://tickets.puppetlabs.com) in the MODULES project and ask to deprecate the old module (and approve the new one if the old one was approved as well).
+* Create a Jira issue at [tickets.puppetlabs.com](https://tickets.puppetlabs.com) in the FORGE project and ask to deprecate the old module (and approve the new one if the old one was approved as well).
+* Do you think the module qualifies to be approved? Wait until it is released, then raise a jira ticket in the [MODULES project][modules] and ask for the `approved` badge.
```

Comment:
@chelnak how would you like this to be raised?

Reply:
if you've a specific template or jira board where we should raise this, let me know!

Reply:
Approval requests go here: 

https://github.com/puppetlabs/puppet-approved-modules